src/tracker/tracker-funcs.jl

```diff
@@ -202,50 +202,50 @@ Return `true` if the floe at `p1` and the floe at `p2` are within a certain dist
 - `t`: tuple of thresholds for elapsed time and distance
 
 """
-function trackercond1(d, delta_time, t1=(dt=(30, 100, 1300), dist=(15, 30, 120)))
-    return (delta_time < t1.dt[1] && d < t1.dist[1]) ||
-           (delta_time >= t1.dt[1] && delta_time <= t1.dt[2] && d < t1.dist[2]) ||
-           (delta_time >= t1.dt[3] && d < t1.dist[3])
+function trackercond1(d, delta_time, search_thresholds=(dt=(30, 100, 1300), dist=(15, 30, 120)))
+    return (delta_time < search_thresholds.dt[1] && d < search_thresholds.dist[1]) ||
+           (delta_time >= search_thresholds.dt[1] && delta_time <= search_thresholds.dt[2] && d < search_thresholds.dist[2]) ||
+           (delta_time >= search_thresholds.dt[3] && d < search_thresholds.dist[3])
 end
 
 """
-    trackercond2(area1, ratios, t2=(area=1200, arearatio=0.28, majaxisratio=0.10, minaxisratio=0.12, convex_area=0.14))
+    trackercond2(area1, ratios, small_floe_settings=(area=1200, arearatio=0.28, majaxisratio=0.10, minaxisratio=0.12, convex_area=0.14))
```

Comment:
Consider renaming `trackercond2` as `large_floe_condition`. @danielmwatkins what do you think?
```suggestion
    trackercond2(area1, ratios, large_floe_settings=(area=1200, arearatio=0.28, majaxisratio=0.10, minaxisratio=0.12, convex_area=0.14))
```